### PR TITLE
Update some functions and add result fields

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**

### CODE REVIEW

File: WithSecure/withsecure/isolate_device_from_network_action.py [hunk 1]
```diff
@@ -8,13 +9,22 @@ class ActionArguments(BaseModel):
...
             parameters["message"] = arguments.message
 
         # execute the operation
-        self._execute_operation_on_device(
+        response = self._execute_operation_on_device(
             operation_name="isolateFromNetwork", target=arguments.target, parameters=parameters
         )
+
+        return RemoteOperationResponse(
+            multistatus=response.get("multistatus"), transactionId=response.get("transactionId")
```
Comment (squioc) [inline WithSecure/withsecure/isolate_device_from_network_action.py:29]: As the properties cannot be null
```suggestion
            multistatus=response.get("multistatus", []), transactionId=response["transactionId"]
```

File: WithSecure/withsecure/isolate_device_from_network_action.py [hunk 2]
```diff
@@ -8,13 +9,22 @@ class ActionArguments(BaseModel):
     message: str | None = None
 
 
+class RemoteOperationResponse(BaseModel):
+    multistatus: list[Any]
+    transactionId: str
```
Comment (squioc) [inline WithSecure/withsecure/isolate_device_from_network_action.py:14]: I suggest outsourcing this declaration in a `models.py` file and import it in this file.

File: WithSecure/withsecure/release_device_from_network_isolation_action.py [hunk 3]
```diff
@@ -7,7 +8,18 @@ class ActionArguments(BaseModel):
     target: str
 
 
+class RemoteOperationResponse(BaseModel):
+    multistatus: list[Any]
+    transactionId: str
```
Comment (squioc) [inline WithSecure/withsecure/release_device_from_network_isolation_action.py:13]: Same as above

**Review by squioc — CHANGES_REQUESTED** [review]
Thanks for the fixes. Some changes to apply

**Review by squioc — APPROVED** [review]
LGTM. Just a question about the way to load the list of Detections.

File: WithSecure/withsecure/list_detections_for_incident.py [hunk 4]
```diff
@@ -7,7 +8,35 @@ class ActionArguments(BaseModel):
...
+
 class ListDetectionForIncident(IncidentOperationAction):
-    def run(self, arguments: ActionArguments) -> None:
+    def run(self, arguments: ActionArguments) -> DetectionItems:
         # execute the operation
-        self._execute_operation_on_incident(operation_name="ListDetectionForIncident", target=arguments.target)
+        detections = self._execute_operation_on_incident(
+            operation_name="ListDetectionForIncident", target=arguments.target
+        )
+        return DetectionItems(detections=detections)
```
[the remark this answers is not in the document]
Comment (TOUFIKIzakarya) [reply WithSecure/withsecure/list_detections_for_incident.py:42]: As `model_validate` was introduced in pydantic V2. So I think we can't use it.

**Pull request merged**